network/src/protocols/mod.rs

```diff
@@ -58,6 +58,7 @@ pub trait CKBProtocolContext: Send {
     fn connected_peers(&self) -> Vec<PeerIndex>;
     fn report_peer(&self, peer_index: PeerIndex, behaviour: Behaviour);
     fn ban_peer(&self, peer_index: PeerIndex, duration: Duration);
+    fn pause_send(&self) -> bool;
```

Comment:
It looks like a method which performs an action to pause sending. `send_paused` is a better name.

Reply:
`pause_send` is from bitcoin, but `send_paused` seems more readable.